frontend/src/__tests__/cypress/cypress/tests/mocked/home/homeResources.cy.ts

```diff
@@ -11,21 +11,24 @@ describe('Home page Resources section', () => {
     homePage.initHomeIntercepts();
     homePage.visit();
   });
+
   it('should show the resources section', () => {
-    cy.findByTestId('landing-page-resources').scrollIntoView();
-    cy.findByTestId('resource-card-create-jupyter-notebook').should('be.visible');
+    const homeResourceSection = homePage.getHomeResourceSection();
+    homeResourceSection.find().scrollIntoView();
+    homeResourceSection.findJupyterCard().should('be.visible');
   });
+
   it('should hide the the resource section if none are available', () => {
     cy.interceptOdh('GET /api/docs', []);
     cy.interceptOdh('GET /api/components', null, []);
     cy.interceptOdh('GET /api/quickstarts', []);
     homePage.visit();
-
-    cy.findByTestId('landing-page-resources').should('not.exist');
+    homePage.getHomeResourceSection().find().should('not.exist');
   });
   it('should navigate to the resources page', () => {
-    cy.findByTestId('goto-resources-link').scrollIntoView();
-    cy.findByTestId('goto-resources-link').click();
-    cy.findByTestId('app-page-title').should('have.text', 'Resources');
+    const homeResourceSection = homePage.getHomeResourceSection();
+    homeResourceSection.findGoToResourceLink().scrollIntoView();
```

Comment:
It should be unnecessary to scoll into view elements prior to taking action:
https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Actionability